src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -65,6 +67,10 @@ module.exports = Model.extend({
         this.fetch();
       }
     }, this);
+
+    this.on('change:localTimezone', function () {
```

Comment:
we found problems with too deep call stack (probably because of the prototype method)